docs/dev/dod.md

```diff
@@ -0,0 +1,16 @@
+# Definition of Done
+These criteria must be met before a feature or artifact can be considered "done".
+
+## Code
+- Code coverage: >75%
+- Codacy grade A
```

Comment:
I think we should keep grade A for now, if it shows to be too much effort, the document can still be adjusted. Right now it seems manageable.